front/pages/w/[wId]/a/index.tsx

```diff
@@ -566,7 +566,7 @@ export default function Developers({
   const router = useRouter();
   const handleTabChange = async (tabId: string) => {
     const query = { ...router.query, t: tabId };
-    await router.push({ query });
+    await router.replace({ query });
```

Comment:
Why replace?

Reply:
For cleaner browser history management - cf https://github.com/dust-tt/dust/pull/4845#pullrequestreview-2022712693

Reply:
Well these tabs are "more engaging" IMHO. And I personally do rely on browser history to move from app to datasets to config etc...